spec/services/record_declarations/param_validation_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+describe "Record Declaration Validation" do
```

Comment:
Should these tests live in `spec/services/record_declarations/retained/early_career_teacher_spec.rb`?

Reply:
well, yeah, I've grouped them here as well as all schema validation tests. Will move them into event specific folders

Reply:
Moved them in 81469b5